app_dart/test/request_handlers/vacuum_github_commits_test.dart

```diff
@@ -164,11 +167,11 @@ void main() {
 
     test('skips commits for which transaction commit fails', () async {
       githubCommits = <String>['2', '3', '4'];
-      config.flutterBranchesValue = <String>['master'];
+      config.flutterBranchesValue = <String>['main'];
 
       /// This test is simulating an existing branch, which must already
       /// have at least one commit in the datastore.
-      final Commit commit = shaToCommit('1', 'master', Config.flutterSlug);
+      final Commit commit = shaToCommit('1', 'main', Config.engineSlug);
```

Comment:
Is this just to avoid this test being disrupted later when flutter/flutter changes default branches?

Reply:
Yep!